src/Contract.php

```diff
@@ -69,7 +69,7 @@ public function sign(DocumentSignInterface $documentSign)
         $document = $documentSign->create(storage_path($this->file), Str::replaceFirst('contracts/', '', $this->file));
         $this->external_id = $document['id'];
 
-        $documentSign->signer('test@live.com')->sign($document['id'], 'Rentbay', 'noreply@rentbay.nl');
+        $documentSign->signer('hidot23263@verumst.com')->sign($document['id'], 'Rentbay', 'noreply@rentbay.nl');
```

Comment:
replace this email for a contract->user->email.
If that relation does not yet exists create it ;-)